purchase_internal_validation/README.rst

```diff
@@ -0,0 +1,71 @@
+============================
```

Comment:
Hi @dufresnedavid Thanks for the contribution.
I miss the badge at beginning of README